src/blocks/recommendations/README.md

```diff
@@ -0,0 +1,66 @@
+# Parse.ly Recommendations Block
```

Comment:
```suggestion
# Parse.ly Related Links Block
```

Should we consider changing the name of the block (& all underlying nomenclature) to match the endpoint? I can see arguments both ways. For example, we may end up composing the results of multiple endpoints to offer recommendations.

Reply:
I personally prefer "Related Links" over "Recommendations" as it is more descriptive/precise of what the block actually does. What do other contributors think about this?